app/views/checkins/_checkins.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= search_form_for q, url: event_url, data: {"controller": "turbo_search", "turbo-frame": "results"} do |f| %>
+<div class="card">
+  <div class="card-header">
+    <h3>Search people</h3>
+  </div>
+  <div class="card-body">
+      <div class="col">
+        <%= f.input :person_display_name_cont, label: 'Name' %>
+      </div>
+  </div>
+  <div class="card-footer">
+    <% if event.locked %>
+      <div class="btn btn-success disabled">Checkin</div>  
+    <% else %>
+      <%= link_to "Checkin", new_event_checkin_path(event), class: 'btn btn-success' %>
+    <% end %>
+
+    <%= f.submit 'Search', class: 'btn btn-primary', data: { action: "turbo_search#search" } %>
```

Comment:
searching for people doesn't work properly. something's wrong with policy_scope ransack